packages/dds/tree/src/core/forest/forest.ts

```diff
@@ -58,7 +58,12 @@ export interface ForestEvents {
  *
  * When invalidating, all outstanding cursors must be freed or cleared.
  */
-export interface IForestSubscription extends Listenable<ForestEvents> {
+export interface IForestSubscription {
+	/**
+	 * Events for this forest.
```

Comment:
For our internal types, I don't actually think it's practically necessary to document the `events` property at either the interface level or the implementation level since it's clear what is going on (and it's the events themselves, not the event emitter, that needs documentation). Nonetheless I added a brief doc comment in places where the surrounding properties also had consistent doc comments.